src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
@timj - @mdboom should have a little bit more insight as the author of #14 .  I *think* the main point was to avoid compiler warnings that were causing problems somewhere.  (And this is the sort of thing we are willing to keep separate between ERFA and SOFA.)

Even if we decide to drop this change, though, the README would need to be updated to reflect this change.